### ISSUE TITLE
2 Bugs + Question

### ISSUE THREAD

**Issue by DominusMaximus** [demilich1/metastone#246]
- If I have 6 minions on board, when playing a 7th minion I'm unable to place that minion at the far left position because of the arrow graphic for that position not being visible:

![minion placement](https://cloud.githubusercontent.com/assets/21368033/18492311/a63f7bf0-79d8-11e6-97b0-77c85f2e694e.jpg)

- Southsea Squidface is missing the ARGH! race tag.

- I have a question concerning the "custom" set option found in the deckbuilder. How do I go about using this? I'm interested in creating a custom set that simply contains all of the Standard cards, so that I may filter out Wild cards when building a Standard deck.

- Kudos on the back button in Play mode. I'll definitely be making use of that.

**Comment by webadict** [comment]
- Yeah, that left button probably won't be fixed for a while. It sucks, but it's a low priority issue.
- Fixed the Pirate tag.
- Near the bottom of the screen in the Deck Builder, there's a search bar and a drop down menu. In the drop down menu, you can choose different Card Sets. It's not the best solution, but we could probably make a deck filter filter, so that you can find all the cards of a certain deck filter (All, Standard, Wild, etc.) It's not hard to implement, it's just about making it not look awful. I'll see what can be done.

**Comment by DominusMaximus** [comment]
Alright, understood.

In regards to the Deck Builder, that drop down menu is what I was referring to. The last set that appears in that menu says "custom". No cards were appearing when it was selected as a filter. Anyways, I just figured out it was pointing to the "set" tag within each card json. I created a folder called "standard" within the metastone/cards folder and copied all of the standard cards to it. I then replaced all of the set tags within those jsons with the "CUSTOM" tag. Now when selecting the custom filter from the drop down menu in Deck Builder, only Standard cards are shown.
